.github/workflows/Tests.yaml

```diff
@@ -225,6 +225,14 @@ jobs:
             fi
           fi
 
+          BUILD_PYTHON_BINDINGS=ON
+
+          if [ ${{ matrix.build_type }} = Release ]; then
+            if [ ${{ matrix.compiler }} = clang-8 ]; then
+              BUILD_PYTHON_BINDINGS=OFF;
+            fi
+          fi
```

Comment:
You can use the `strategy.matrix.include` configuration on line 166 to achieve what these `if` statements do (see [these docs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#example-including-configurations-in-a-matrix-build)). I think all that needs to be done is this:
- Add this item to the list on line 166:
    ```yaml
    - compiler: clang-8
      build_type: Release
      BUILD_PYTHON_BINDINGS: OFF
    ```
- In line 228 you can use the variable in the bash script:
    ```bash
    BUILD_PYTHON_BINDINGS=${{ matrix.BUILD_PYTHON_BINDINGS }}
    ```
- The variable is empty for the other configurations, so you can default it to `'ON'` in the `cmake` call:
    ```
    -D BUILD_PYTHON_BINDINGS=${BUILD_PYTHON_BINDINGS:-'ON'}
    ```
When I'm back, I'll expand this configuration to {Py2, Py3, Off} spread across the compilers.